HousingRepairsOnlineApi.Tests/UseCasesTests/SendInternalEmailUseCaseTests.cs

```diff
@@ -0,0 +1,186 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using FluentAssertions;
+using HousingRepairsOnlineApi.Gateways;
+using HousingRepairsOnlineApi.UseCases;
+using Moq;
+using Xunit;
+
+namespace HousingRepairsOnlineApi.Tests.UseCasesTests
+{
+    public class SendInternalEmailUseCaseTests
+    {
+        private readonly Mock<INotifyGateway> govNotifyGatewayMock;
+        private readonly SendInternalEmailUseCase systemUnderTest;
+
+        public SendInternalEmailUseCaseTests()
+        {
+            govNotifyGatewayMock = new Mock<INotifyGateway>();
+            systemUnderTest = new SendInternalEmailUseCase(govNotifyGatewayMock.Object, "templateId", "dr.who@tardis.com");
+        }
+
+        //Arrange
```

Comment:
Did you mean to have this "Arrange" comment on this test data method?
Without context of the test it is for, don't think it would be relevant to have the comment.

It could have been part of `GivenAnInvalidBookingRef_WhenExecute_ThenExceptionIsThrown` originally as that no longer has an "Arrange".

Multiple occurrences of the same in this file.